config_examples/arm/CentOS_7.1.json

```diff
@@ -29,6 +29,7 @@
             "ssh_username": "{{user `ssh_user`}}",
             "ssh_password": "{{user `ssh_pass`}}",
 
+            "os_type": "linux",
```

Comment:
should not be necessary if you're deploying from a stock image. PIR has this information?